activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Add database_exists? method to connection adapters to check if a database exists.
+
+      *Guilherme Mansur*
```

Comment:
@robertomiranda : The indentation is wrong here. It looks perfect on the markdown tho.

https://github.com/rails/rails/blob/6-0-stable/activerecord/CHANGELOG.md